web/client/components/mapcontrols/mouseposition/CRSSelector.jsx

```diff
@@ -41,22 +38,19 @@ let CRSSelector = React.createClass({
             }
         }
         return (
-              (this.props.enabled) ? (<Input
+              (this.props.enabled) ? (<select
```

Comment:
the Input control works with the react-bootstrap version we include in MS2. Can you give more details on the use case where it does not work for you?